### PR TITLE
Log task outputs

### PR DESCRIPTION
**Pull request by romasku**
Closes https://github.com/neuromation/neuro-flow/issues/61

### CODE REVIEW

**Comment by romasku** [comment]
Do we need some click option to turn off this behaviour?

File: neuro_flow/batch_runner.py [hunk 1]
```diff
@@ -266,15 +266,21 @@ async def _process_started(
...
                     )
                     str_status = fmt_status(finished[st.id].status)
                     raw_id = fmt_raw_id(st.raw_id)
-                    click.echo(f"Task {str_full_id} [{raw_id}] is {str_status}")
+                    click.echo(
+                        f"Task {str_full_id} [{raw_id}] is {str_status}"
+                        + (" with following outputs:" if fd.outputs else "")
+                    )
+                    for key, value in fd.outputs.items():
+                        click.echo(f"f{key}: {value}")
```
Comment (asvetlov) [inline neuro_flow/batch_runner.py:282]:
```suggestion
                        click.echo(f"  {key}: {value}")
```

File: neuro_flow/batch_runner.py [hunk 2]
```diff
@@ -291,14 +297,20 @@ async def _process_started(
...
                     outputs,
                 )
                 str_status = fmt_status(finished[st.id].status)
-                click.echo(f"Action {str_full_id} is {str_status}")
+                click.echo(
+                    f"Action {str_full_id} is {str_status}"
+                    + (" with following outputs:" if fd.outputs else "")
+                )
+                for key, value in fd.outputs.items():
+                    click.echo(f"f{key}: {value}")
```
Comment (asvetlov) [inline neuro_flow/batch_runner.py:312]:
```suggestion
                    click.echo(f"  {key}: {value}")
```

**Comment by asvetlov** [comment]
> Do we need some click option to turn off this behavior?

Don't know, it is @mariyadavydova request
I can imagine when long outputs screw up the log.
Ideally, we should support `-q/--quiet` and `-v/--verbose` flags as we do in `neuro` CLI already.

**Comment by mariyadavydova** [comment]
I wouldn't bother about long output, at least for now. I expect them to be some calculation values or data/artifacts paths, which are more or less short. Let's merge this PR.

**Review by asvetlov — APPROVED** [review]
Ok, let's merge after the conflict resolution.

**Pull request merged**